### ISSUE TITLE
path type throws an exception on non-existent paths in v0.94.0

### ISSUE THREAD

**Issue by saketh21v** [nushell/nushell#13003]
### Describe the bug

`path type` throws an exception on non-existent paths instead of returning an empty string.

### How to reproduce

1. Run `"non-existent-path" | path type`
2. Throws an exception

### Expected behavior

According to `help path type`, `path type` should return an empty string when the path doesn't exist
```
This checks the file system to confirm the path's object type.
If nothing is found, an empty string will be returned.
```

### Screenshots

<img width="703" alt="Screenshot 2024-05-30 at 12 21 35 PM" src="https://github.com/nushell/nushell/assets/9252678/6e649afb-0454-49ef-83a0-cdddd2be1375">

<img width="1109" alt="Screenshot 2024-05-30 at 12 21 53 PM" src="https://github.com/nushell/nushell/assets/9252678/91d6f331-7efc-4c44-a9ce-1b658fc7cb13">

### Configuration

| key                | value                                           |
| ------------------ | ----------------------------------------------- |
| version            | 0.94.0                                          |
| major              | 0                                               |
| minor              | 94                                              |
| patch              | 0                                               |
| branch             |                                                 |
| commit_hash        |                                                 |
| build_os           | macos-aarch64                                   |
| build_target       | aarch64-apple-darwin                            |
| rust_version       | rustc 1.78.0 (9b00956e5 2024-04-29) (Homebrew)  |
| cargo_version      | cargo 1.78.0                                    |
| build_time         | 2024-05-28 19:04:09 +00:00                      |
| build_rust_channel | release                                         |
| allocator          | mimalloc                                        |
| features           | default, sqlite, system-clipboard, trash, which |
| installed_plugins  | highlight                                       |

**Comment by fdncred** [comment]
Agreed that it works differently now and that it doesn't do what the help text says. I'm not quite sure what we're going to do about this yet, if anything.

**Comment by fdncred** [comment]
closed by https://github.com/nushell/nushell/pull/13006
